requirements.txt

```diff
@@ -1,10 +1,10 @@
-psycopg2-binary==2.7.4
-torch==0.4.0
+psycopg2-binary==2.7.7
+torch==1.0.0
```

Comment:
Torch 1.0.0 here should be fine but I remember some discussion regarding it's stability @thodrek @Ihabilyas